website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -6,14 +6,22 @@ sidebar_label: "MetricFlow time spine"
 tags: [Metrics, Semantic Layer]
 ---
 
-It's common in analytics engineering to have a date dimension or "time spine" table as a base table for different types of time-based joins and aggregations. The structure of this table is typically a base column of daily or hourly dates, with additional columns for other time grains, like fiscal quarter, defined based on the base column. You can join other tables to the time spine on the base column to calculate metrics like revenue at a point in time, or to aggregate to a specific time grain.
+It's common in analytics engineering to have a date dimension or "time spine" table as a base table for different types of time-based joins and aggregations. The structure of this table is typically a base column of daily or hourly dates, with additional columns for other time grains, like fiscal quarters, defined based on the base column. You can join other tables to the time spine on the base column to calculate metrics like revenue at a point in time, or to aggregate to a specific time grain.
 
-MetricFlow requires you to define a time spine table as a project level configuration, which then is used for various time-based joins and aggregations, like cumulative metrics. At a minimum, you need to define a time spine table for a daily grain. You can optionally define a time spine table for a different granularity, like hourly.
+MetricFlow requires you to define a time spine table as a project-level configuration for time-based joins and aggregations, like cumulative metrics. 
+- You only need to configure time spine models that the Semantic Layer should recognize.
+- At a minimum, define a time spine table for a daily grain.
+- You can optionally define a time spine table for a different granularity, like hourly.
+- Note that if you don’t have a date or calendar model in your project, you'll need to create one. 
```

Comment:
```suggestion
- Note that if you don’t have a date or calendar model in your project, you'll need to create one. 
- If you're looking to specify the grain of a time dimension so that MetricFlow can transform the underlying column to the required granularity, refer to the [Time granularity documentation](/docs/build/dimensions?dimension=time_gran)
```